WebRoot/messages/ZmMsg.properties

```diff
@@ -4222,6 +4222,8 @@ COLUMN_WIDTH_TYPE_CN = 20
 
 # filter rules and zimlet list
 COLUMN_WIDTH_ACTIVE = 50
+COLUMN_WIDTH_ZIMLET_ACTIVE = 65
```

Comment:
instead of creating new property, can we change value of COLUMN_WIDTH_ACTIVE that will generically change in all other list views